frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -974,6 +974,9 @@ export const ClusterServiceVersionDetails: React.FC<ClusterServiceVersionDetails
   const csvPlugins = getClusterServiceVersionPlugins(metadata?.annotations);
   const subscription = subscriptionForCSV(props.subscriptions, props.obj);
 
+  let permissions = _.get(spec, 'install.spec.permissions', []);
+  permissions = _.uniqBy(permissions, 'serviceAccountName');
```

Comment:
I'd suggest

```suggestion
  const permissions = _.uniqBy(spec?.install?.spec?.permissions, 'serviceAccountName');
```

I believe `_.uniqBy` will return an empty array when the first parameter is `undefined`.

Reply:
Confirmed this works the same as what I have proposed. I will make this change 👍

Reply:
Done!